frontend/src/pages/SubProjects/SubProjectContainer.js

```diff
@@ -132,6 +132,7 @@ class SubProjectContainer extends Component {
                 subProjects={this.props.filteredSubProjects}
                 highlightingRegex={this.props.highlightingRegex}
                 isDataLoading={this.props.isDataLoading}
+                idsPermissionsUnassigned={this.props.idsPermissionsUnassigned}
```

Comment:
delete this prop from this container including all child components since it is not used.

Reply:
it is passed to <Subprojects> and then passed to <SubProjectTable>.
To make it more transparent, i added this prop explicitly to the component.